test/test_va_api_create_image.cpp

```diff
@@ -0,0 +1,53 @@
+ /*
+ * Copyright (C) 2016 Intel Corporation. All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a
+ * copy of this software and associated documentation files (the
+ * "Software"), to deal in the Software without restriction, including
+ * without limitation the rights to use, copy, modify, merge, publish,
+ * distribute, sub license, and/or sell copies of the Software, and to
+ * permit persons to whom the Software is furnished to do so, subject to
+ * the following conditions:
+ *
+ * The above copyright notice and this permission notice (including the
+ * next paragraph) shall be included in all copies or substantial portions
+ * of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+ * OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT.
+ * IN NO EVENT SHALL PRECISION INSIGHT AND/OR ITS SUPPLIERS BE LIABLE FOR
+ * ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+#include "test_va_api_create_image.h"
+
+namespace VAAPI {
+VAAPICreateImage::VAAPICreateImage ()
+{
+    m_vaDisplay = doInitialize();
+}
+
+VAAPICreateImage::~VAAPICreateImage()
+{
+    doTerminate();
+}
+
+TEST_P(VAAPICreateImage, CreateImageTest)
+{
+    uint32_t currentFourcc = ::testing::get<0>(GetParam());
+    std::pair<uint32_t, uint32_t> currentResolution
```

Comment:
make this a const reference variable to avoid copy

Reply:
Fixed.